fl4health/utils/losses.py

```diff
@@ -11,7 +11,7 @@ class Losses:
     def __init__(
         self,
         checkpoint: torch.Tensor,
-        backward: torch.Tensor,
+        backward: Union[torch.Tensor, Dict[str, torch.Tensor]],
```

Comment:
Backward loss is a dictionary in the case of ensembles.

Reply:
This could be added as a code comment to explain the `Union`.

Reply:
Added a comment and better overall documentation!